Configuration/Settings.yaml

```diff
@@ -2,6 +2,8 @@ Networkteam:
   Neos:
     ContentApi:
       recursiveReferencePropertyDepth: 1
+      documentsFilter:
```

Comment:
As discussed: we should name the setting documentList.ignoredNodeTypes to make it obvious, that this only affects the listing.